mono/mini/ee.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+#include <config.h>
+#include <mono/metadata/metadata.h>
+#include <mono/metadata/object.h>
+#include <mono/utils/mono-compiler.h>
+#include <mono/utils/mono-error.h>
+#include <mono/utils/mono-publib.h>
+#include <mono/eglib/glib.h>
+
+#ifndef __MONO_EE_H__
+#define __MONO_EE_H__
+
+#define MONO_EE_API_VERSION 0x1
+
+typedef struct _MonoInterpStackIter MonoInterpStackIter;
+
+/* Needed for stack allocation */
+struct _MonoInterpStackIter {
+	gpointer dummy [8];
+};
+
+typedef gpointer MonoInterpFrameHandle;
+
+struct _MonoEECallbacks {
```

Comment:
From the IL2CPP side, this looks like what we need to more easily integrate IL2CPP on the Mono runtime. Thanks!